CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add the initial support of cryptographic module loading in SGX enclaves. Refer to the [design document](docs/DesignDocs/CryptoModuleLoadingSupport.md) for more detail.
 - Add the support of getrandom libc API and syscall in enclaves.
+- Add `libsgx-quote-ex`, `sgx-aesm-service` and several SGX AESM plugins to Ansible scripts, which are used to install project requirements.
```

Comment:
Can you also add a link to the attestation out of proc/in proc attestation. Is there a recommendation for what developers  should choose?

Reply:
Done. Out-of-process call path will be the default choice. More information is in the design doc and the attestation sample's README.

Reply:
Thanks Ryan.